<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rangeform</class>
 <widget class="QWidget" name="rangeform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Введите значения</string>
   </property>
   <widget class="QLineEdit" name="lastLineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="firstLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Нижняя граница   --&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Верхняя граница   --&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstLineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
